crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -14,6 +14,10 @@ use crate::prelude::*;
 
 #[derive(Eq, PartialEq, Debug, Default)]
 pub enum TupleParentheses {
+    /// TODO(cnpryer): Why not `Never`?
```

Comment:
Originally I wanted to do `TupleParentheses::Expr(Parentheses::Never)` but I'd need more to handle
```
# Input
h3 = 1, "qweiurpoiqwurepqiurpqirpuqoiwrupqoirupqoirupqoiurpqiorupwqiourpqurpqurpqurpqurpqurpqurüqurqpuriq"

# Ruff
h3 = 1, "qweiurpoiqwurepqiurpqirpuqoiwrupqoirupqoirupqoiurpqiorupwqiourpqurpqurpqurpqurpqurpqurüqurqpuriq"

# Black
+h3 = (
+    1,
+    "qweiurpoiqwurepqiurpqirpuqoiwrupqoirupqoirupqoiurpqiorupwqiourpqurpqurpqurpqurpqurpqurüqurqpuriq",
+)
```
I also feel like I'm misunderstanding what `Some(Parentheses)` implies.

Reply:
https://github.com/astral-sh/ruff/pull/5810#discussion_r1267439876